src/app/code/community/Zendesk/Zendesk/Helper/Sync.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: o5k4r1n
+ * Date: 29-05-18
+ * Time: 10:44 AM
+ */
+class Zendesk_Zendesk_Helper_Sync extends Mage_Core_Helper_Abstract {
+
+    public function getCustomerData($customer){
+        if(!Mage::getStoreConfig('zendesk/general/customer_sync'))
+            return;
+
+        $user = null;
+        //$customer = $event->getCustomer();
+        $email = $customer->getEmail();
+        $orig_email = $customer->getOrigData();
+        $orig_email = $orig_email['email'];
+        echo "correo: ".$email;
```

Comment:
Is this debug code? if so, please remove thanks.

Reply:
Yes it was a debug code, Dropping now